opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1603,12 +1604,14 @@ namespace Opm {
     public:
         int ebosCompToFlowPhaseIdx( const int compIdx ) const
         {
+            assert(compIdx < 3);
             const int compToPhase[ 3 ] = { Oil, Water, Gas };
             return compToPhase[ compIdx ];
         }
 
         int flowToEbosPvIdx( const int flowPv ) const
         {
+            assert(flowPv < 3);
```

Comment:
this assertion will be violated once the solvent or the polymer extension are implemented?